packages/adminrouter/extra/src/test-harness/tests/test_generic.config.yml

```diff
@@ -74,6 +74,15 @@ endpoint_tests:
     type:
       - master
   - tests:
+      is_upstream_req_ok:
```

Comment:
It isn't clear to me what this is testing. Please add a comment explaining what the purpose of this test is. If you feel that nothing could be added that isn't said in the test, ping me on Slack.